cgi/user.pl

```diff
@@ -38,9 +38,28 @@
 use Storable qw/dclone/;
 use Log::Any qw($log);
 
+use Template;
+use Data::Dumper;
+
 my $type = param('type') || 'add';
 my $action = param('action') || 'display';
 
+
+# Initialize the Template module
+my $tt = Template->new({
```

Comment:
I noticed that the initialisation of `$tt` and `$template_data_ref` is now done several times in Display.pm and some .pl files in the same way. Would it make sense to introduce a new `ProductOpener::Template` module with methods that know how to do just that? It could reduce duplicated code and avoid errors.

Reply:
@hangy : maybe we can just make the $tt in Display.pm an exported variable. Then it's initialize once at module startup, and that's it.